pmm/pmm2-ui-tests.groovy

```diff
@@ -147,6 +147,7 @@ pipeline {
                 sauce('SauceLabsKey') {
                     sauceconnect(options: '', sauceConnectPath: '') {
                         sh """
+                            export INSTANCE_IP=${SERVER_IP}
```

Comment:
@puneet0191  maybe it is better to do it once in env section ... https://github.com/Percona-Lab/jenkins-pipelines/blob/2ff14c75f803c01c111f1ed97509da88f8988230/pmm/pmm2-ui-tests.groovy#L36 section ...

Reply:
I have re-used existing env variable in tests, so no need to initiate it in Env. section also, it is automatically done now.